articles/introducing-cross-platform-script-execution.md

```diff
@@ -0,0 +1,59 @@
+![Fleet MDM Cover](../website/assets/images/articles/introducing-cross-platform-script-execution-800x450@2x.png)
+
+At Fleet, we strive to enhance the functionality of our platform, offering more power and flexibility to IT administrators and security teams. Today, we are thrilled to announce an addition to our toolkit - the ability to [execute a shell script](https://fleetdm.com/docs/using-fleet/scripts) via command line (CLI) and Fleet's API across macOS, Windows, and Linux.
+
+This feature, available exclusively in Fleet Premium, unlocks the potential to execute a script on a specific host, empowering professionals to remediate issues or collect logs quickly. Whether using Bourne shell or zsh for Mac and Linux or PowerShell for Windows, this functionality offers a streamlined and efficient solution to tackle challenges head-on.
```

Comment:
```suggestion
This feature, available exclusively in Fleet Premium, unlocks the potential to execute a script on a specific host, empowering professionals to remediate issues or collect logs quickly. Whether using shell scripts on Mac and Linux or PowerShell for Windows, this functionality offers a streamlined and efficient solution to tackle challenges head-on.
```

I think we shouldn't mention Bourne and zsh here so folks don't confused about whether they have the option. Just keep it to "shell." We explain it's default shell below.

Reply:
Do you agree?

Reply:
I'm about to log off for the weekend. Please feel free to dismiss my review if it's blocking the merge!